corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -134,7 +134,9 @@ hqDefine('cloudcare/js/backbone/apps.js', function () {
             return this.get("urlRoot");
         },
         getSubmitUrl: function () {
-            return this.get('post_url');
+            // http://stackoverflow.com/a/2599721/835696
```

Comment:
We might want to test this on https before deploying it. Have a vague
recollection the base URL thing was related to that
On Jun 16, 2016 7:54 PM, "Ben" <notifications@github.com> wrote:

> In corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js
> <https://github.com/dimagi/commcare-hq/pull/12089#discussion_r67392898>:
>
> > @@ -134,7 +134,9 @@ hqDefine('cloudcare/js/backbone/apps.js', function () {
> >              return this.get("urlRoot");
> >          },
> >          getSubmitUrl: function () {
> > -            return this.get('post_url');
> > +            // http://stackoverflow.com/a/2599721/835696
>
> this wasn't a super highly voted answer, but i think it works fine for us.
> essentially captures anything that isn't a / and deletes it
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/12089/files/59a06d3d6776ceba188a58c05feb11e8506fbcaf#r67392898>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAED-8XLhh04K1wg22wHTm1vCrb7taOgks5qMY3IgaJpZM4I3pCe>
> .
>